tests/CMakeLists.txt

```diff
@@ -50,12 +60,14 @@ endfunction(parsec_addtest_executable)
 # name in that case.
 #
 function(parsec_addtest_cmd target)
```

Comment:
This looks like a very unexpected side effect (aka. the macro does not do anything if parsec is not the main project). I would rather create a special testing configuration for parsec, and add all our tests in there. The configuration does not have to be activated by the upstream project.

Reply:
After looking at the documentation for CMake `add_test`, I think the better option is to keep doing `add_test` unconditionally, and TTG's CMake, should define `BUILD_TESTING` to OFF when configuring parsec. I removed the conditional thing for this function.